libsoftwarecontainer/include/softwarecontainer.h

```diff
@@ -103,7 +103,7 @@ class SoftwareContainer :
 
     std::shared_ptr<ContainerAbstractInterface> m_container;
     pid_t m_pcPid = INVALID_PID;
-    std::vector<std::unique_ptr<Gateway> > m_gateways;
+    std::vector< std::unique_ptr<Gateway> > m_gateways;
```

Comment:
I think we should not use spaces like this, the original space at the end was incorrect.